saleor/graphql/core/utils/__init__.py

```diff
@@ -108,3 +108,15 @@ def get_duplicates_ids(first_list, second_list):
 def get_duplicated_values(values):
     """Return set of duplicated values."""
     return {value for value in values if values.count(value) > 1}
+
+
+def validate_and_strip_field_str_value(cleaned_input, field_name: str):
```

Comment:
A detail, but maybe `validate_required_string_field` would sound more universal?

Reply:
Yes, I was thinking about it but it's not exactly the case here. Field `name` is required to create mutation and optional for the update.
`not_empty` or other combinations are also not the best IMO. I tried to connect validation and the exact action here. But I am not happy either with the result.
(Naming is hard sometimes)

Reply:
The difference between updating and creating is handled by the schema where the field is required or not. Another option is to move `if field_name in cleaned_input` outside and in the function assume that `cleaned_input["value"]` is present? 

We have the same logic in `OrderAddNote` and we should reuse this function there as well.

Reply:
Ok I will reuse it there.